pages/templates/wagtailadmin/login.html

```diff
@@ -1,6 +1,16 @@
-{% overextends "wagtailadmin/login.html" %}
+{% extends "wagtailadmin/login.html" %}
 {% load static %}
 
 {% block branding_login %}
     <img src="/static/images/openstax.png" alt="Openstax" width="80" /> Login to OpenstaxCMS
-{% endblock %}
\ No newline at end of file
+{% endblock %}
+
+{% block furniture %}
+{{ block.super }}
+    {% if user.is_authenticated %}
+        {% include "logout.html" %}
+    {% else %}
+        {% include "login.html" %}
```

Comment:
This is confusing - can we change this to something like oauth_login.html? 
It's being included in a file called login.html.